runner/lib/helpers/buffer-line-transform.js

```diff
@@ -0,0 +1,95 @@
+/* global Buffer */
+/* eslint-disable no-underscore-dangle,no-nested-ternary,no-plusplus */
+
+import { Transform } from 'stream';
+
+export default class BufferLineTransform extends Transform {
+  /**
+   * The BufferLineTransform is reading String or Buffer content from a Readable stream
+   * and writing each line as a Buffer in object mode
```

Comment:
Some optimizations for both human and machine readers are more obvious if you pick either string or bytes and not generalize both.

Reply:
Node streams are usually built to handle both string and buffer, so since it was literally a single line to accept a string, I supported it.

Reply:
A noteworthy design flaw of Node.js 😉